Bio/PDB/FragmentMapper.py

```diff
@@ -202,8 +190,7 @@ def _make_fragment_list(pp, length):
 
 
 def _map_fragment_list(flist, reflist):
-    """
-    Map all frgaments in flist to the closest
+    """Map all frgaments in flist to the closest
```

Comment:
I'd keep the existing text as the main body and insert a one line summary like ``Map flist fragments to closest entry in reflist (PRIVATE).``